pow/src/main/java/tech/pegasys/teku/pow/Eth1StatusLogger.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.pow;
+
+import static tech.pegasys.teku.logging.StatusLogger.STATUS_LOG;
+
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Eth1StatusLogger {
+  private final AtomicBoolean timerActive = new AtomicBoolean();
+  private final AtomicInteger failCount = new AtomicInteger(0);
+  private final long interval;
+  private final TimerTask timerTask;
+  private Timer timer;
+
+  public Eth1StatusLogger(final long interval) {
+    this.interval = interval;
+    this.timerTask =
+        new TimerTask() {
+          @Override
+          public void run() {
+            if (failCount.get() > 1) {
+              STATUS_LOG.eth1ServiceDown(Eth1StatusLogger.this.interval);
```

Comment:
If we have a single request fail and everything else work, this will wind up reporting that the ETH1 service has been down for the entire timer interval which is misleading.  We really want to ignore individual errors but report when there is problem for a sustained period.  If the errors persist for more than one interval we'd also want the time reported in the logs to increase so each log reports the total outage period, not just repeatedly report the same period.

So I'd say we nee to report when requests start working to this class.  Then it can track how long requests have been failing for and alert as needed.  Having a `ErrorTrackingEth1Provider` which implements `Eth1Provider` and delegates much like `ThrottlingEth1Provider` does is probably the neatest way to do this - then it automatically tracks all the ETH1 requests we make and all the error tracking is kept in one place.